packages/protocol/contracts/stability/StableTokenRegistry.sol

```diff
@@ -0,0 +1,106 @@
+pragma solidity ^0.5.13;
```

Comment:
Don't forget to add the migration for this (and use it from the cUSD, cEUR and cREAL migrations)